client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -670,6 +904,29 @@ class CancelPurchaseForm extends React.Component {
 			isJetpackProductSlug( purchase.productSlug ) || isJetpackPlanSlug( purchase.productSlug );
 		const productName = isJetpack ? translate( 'Jetpack' ) : translate( 'WordPress.com' );
 
+		if ( surveyStep === steps.FEEDBACK_STEP ) {
+			return (
+				<>
+					<FormattedHeader
+						brandFont
+						headerText={ translate( 'Your thoughts are needed' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Your thoughts are needed.' )` *ES Score: 12*